charts/osm/templates/preset-mesh-root-certificate.yaml

```diff
@@ -1,4 +1,3 @@
-{{- if .Values.osm.featureFlags.enableMeshRootCertificate }}
```

Comment:
We still need this right?

Reply:
We need the preset-mesh-root-certificate applied regardless of whether or not the used enables mesh root certificate rotation. If enableMeshRootCertificate was false initially false and the user then updated the flag to true, on restart the osm-bootstrap would need the preset-mesh-root-certificate. Our system is not currently designed to retroactively apply the preset-mesh-root-certificate configMap.